python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -564,13 +573,12 @@ def GPIProxyObjectFactory(_obj):
     if hasattr(_obj, proxyObject):
         return getattr(_obj, proxyObject)
 
-    if not hasattr(_obj.__class__, proxyClass):
-        from Ganga.Core.exceptions import GangaException
-        raise GangaException("%s is NOT a Proxyable object" % type(_obj))
+    ## This is defined within Objects.py, we could probably store this elsehere
+    ## (We probably do) but as this is guaranteed to be accessible for GangaObjects I will use this
```

Comment:
*Very* minor point: PEP8 [suggests](https://www.python.org/dev/peps/pep-0008/#block-comments) using a single `#` and a space for block comments.